app/views/validators/_block_history_chart.html.erb

```diff
@@ -9,31 +9,50 @@
 
     // The data for our dataset
     data: {
-        // x-axis labels
-        labels: <%= @data.keys %>,
-        // An Array of JSON hashes for each dataset to plot. The y-axis is
-        // dynamicly calculated. Look for othe options to control the y-axis if
-        // desired.
-        datasets: [
-          {
-            label: 'Skip %',
-            fill: false,
-            backgroundColor: 'Green',
-            borderColor: 'Green',
-            borderWidth: 1,
-            radius: 0,
-            data: <%= @data.map { |k,v| v[:skipped_slot_percent].round(1) }%>
-          },
-          {
-            label: 'Cluster %',
-            fill: false,
-            backgroundColor: 'Blue',
-            borderColor: 'Blue',
-            borderWidth: 1,
-            radius: 0,
-            data: <%= @data.map { |k,v| v[:cluster_skipped_slot_percent].round(1) } %>
-          }
-        ]
+      // x-axis labels
+      labels: <%= @data.keys %>,
+      // An Array of JSON hashes for each dataset to plot. The y-axis is
+      // dynamicly calculated. Look for othe options to control the y-axis if
+      // desired.
+      datasets: [
+        {
+          label: 'Skip %',
+          fill: false,
+          backgroundColor: 'Green',
+          borderColor: 'Green',
+          borderWidth: 1,
+          radius: 0,
+          data: <%= @data.map { |k,v| v[:skipped_slot_percent].round(1) }%>
+        },
+        {
```

Comment:
Maybe we can drop this line from the chart -- just show the 'Cluster Moving Avg Skip %'

Reply:
Sure, I'll just comment out for now so that we can add back later if wanted.